config/manager/manager.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    control-plane: controller-manager
+    control-plane: openstack-controller-manager
```

Comment:
I thought the idea was we could use this to search for all the operator controllers. If each operator has an individual name here wouldn't that change our use case here?

Reply:
okay. I see in the commit message now this is strictly required to make webhooks work across all operators. we'll need to adopt similar label updates across the board then

Reply:
Also see this was discussed on the glance-operator. so ignore me here